changelogs/fragments/492-vmware_resource_pool-esxi_host.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
@mariolenz Can you make this as `breaking_changes` since `cluster` is not a required parameter? This will notify to the user via changelog entry.